Won't compile: `libsmartcalc` doesn't exist

I tried to test this project but it looks like the dependancy `libsmartcalc` no longer exists, this is the output I get:
```
❯ cargo install --path .
  Installing smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)
    Updating crates.io index
    Updating git repository `https://github.com/erhanbaris/smartcalc`
error: failed to compile `smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)`, intermediate artifacts can be found at `/home/user/projects/smartcalc-tui/target`

Caused by:
  no matching package found
  searched package name: `libsmartcalc`
  perhaps you meant:      smartcalc
  location searched: https://github.com/erhanbaris/smartcalc
  required by package `smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)`
```
replacing `libsmartcalc` with `smartcalc` produces the following errors:

```
❯ cargo install --path .
  Installing smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)
    Updating crates.io index
    Updating git repository `https://github.com/erhanbaris/smartcalc`
   Compiling smartcalc v1.0.4 (https://github.com/erhanbaris/smartcalc#507ed954)
   Compiling smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)
error[E0432]: unresolved import `smartcalc::executer`
  --> src/lib.rs:55:28
   |
55 |             use smartcalc::executer::initialize;
   |                            ^^^^^^^^ could not find `executer` in `smartcalc`

error[E0603]: module `app` is private
  --> src/lib.rs:54:28
   |
54 |             use smartcalc::app::{ExecuteLine, ExecuteLineResult, SmartCalc};
   |                            ^^^ private module
   |
note: the module `app` is defined here
  --> /home/user/.cargo/git/checkouts/smartcalc-d93f0c50fe48d91c/507ed95/src/lib.rs:26:1
   |
26 | pub(crate) mod app;
   | ^^^^^^^^^^^^^^^^^^^

warning: ignoring -C extra-filename flag due to -o flag

Some errors have detailed explanations: E0432, E0603.
For more information about an error, try `rustc --explain E0432`.
warning: `smartcalc-tui` (lib) generated 1 warning
error: failed to compile `smartcalc-tui v1.0.1 (/home/user/projects/smartcalc-tui)`, intermediate artifacts can be found at `/home/user/projects/smartcalc-tui/target`

Caused by:
  could not compile `smartcalc-tui` due to 2 previous errors; 1 warning emitted

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,9 +17,12 @@ colored = "2.0.0"
 linefeed = "0.6.0"
 crossbeam-channel = "0.5.1"
 parking_lot = "0.11.1"
-libsmartcalc = { git = "https://github.com/erhanbaris/smartcalc" }
+smartcalc = "1.0.4"
 num-format = { version = "0.4.0", features = ["with-system-locale"] }
 const_format = "0.2.19"
+localzone = "0.2.0"
 
 # disable logging so it doesn't interrupt tui
 log = { version = "0.4.14", features = ["max_level_off", "release_max_level_off"] }
+chrono = "0.4.19"
+chrono-tz = "0.5.3"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -7,7 +7,7 @@ use termion::input::TermRead;
 
 const LANG: &'static str = "en";
 
-// TODO: display libsmartcalc version as well as smartcalc-tui
+// TODO: display smartcalc version as well as smartcalc-tui
 // (probably want built::util::parse_versions https://docs.rs/built/latest/built/)
 const VERSION: &'static str = env!("CARGO_PKG_VERSION");
 
@@ -51,16 +51,27 @@ pub fn spawn() -> Result<(), Box<dyn std::error::Error>> {
     {
         let ps = Arc::downgrade(&prompt.state());
         std::thread::spawn(move || {
-            use libsmartcalc::app::{ExecuteLine, ExecuteLineResult, SmartCalc};
-            use libsmartcalc::executer::initialize;
+            use chrono::{Local, TimeZone};
+            use chrono_tz::{OffsetName, Tz};
             use num_format::SystemLocale;
+            use smartcalc::SmartCalc;
 
-            initialize();
+            let timezone = match localzone::get_local_zone() {
+                Some(tz) => match tz.parse::<Tz>() {
+                    Ok(tz) => {
+                        let dt = Local::today().naive_local();
+                        tz.offset_from_utc_date(&dt).abbreviation().to_string()
+                    }
+                    Err(_) => "UTC".to_string(),
+                },
+                None => "UTC".to_string(),
+            };
 
             let mut app = SmartCalc::default();
             let locale = SystemLocale::default().unwrap();
-            app.config.decimal_seperator = locale.decimal().to_string();
-            app.config.thousand_separator = locale.separator().to_string();
+            app.set_decimal_seperator(locale.decimal().to_string());
+            app.set_thousand_separator(locale.separator().to_string());
+            app.set_timezone(timezone).unwrap();
 
             loop {
                 match ps.upgrade() {
@@ -76,11 +87,11 @@ pub fn spawn() -> Result<(), Box<dyn std::error::Error>> {
                         // drop(ps) then relock after execution?
                         let res = app.execute(LANG, ps.input.clone());
 
-                        match res.lines[0] {
-                            Some(ExecuteLine {
-                                result: Ok(ExecuteLineResult { ref output, .. }),
-                                ..
-                            }) => ps.set_hint(output),
+                        match &res.lines[0] {
+                            Some(line) => match &line.result {
+                                Ok(result) => ps.set_hint(&result.output),
+                                _ => ps.clear_hint(),
+                            },
                             _ => ps.clear_hint(),
                         }
                     }

